packages/slice-machine/package.json

```diff
@@ -88,7 +88,6 @@
     "semver": "^7.3.2",
     "serve-static": "^1.14.1",
     "slash": "3.0.0",
-    "sm-commons": "^0.0.22",
```

Comment:
guess that's why the package-lock has been modifed.

Reply:
yes :)